bgpd/bgp_route.c

```diff
@@ -7416,6 +7426,32 @@ bgp_show_table (struct vty *vty, struct bgp *bgp, struct bgp_table *table,
                 if (! community_list_exact_match (ri->attr->community, list))
                   continue;
               }
+	    if (type == bgp_show_type_community_all)
```

Comment:
`type == bgp_show_type_community_all` is not lcommunity-related and already handled ~30 lines above, seems it accidentally go duplicated.

I'll remove it with the whitespace cleanups.